test/init.bats

```diff
@@ -37,7 +37,7 @@ create_executable() {
 
 @test "setup shell completions" {
   exec_root="$(cd $BATS_TEST_DIRNAME/.. && pwd)"
-  run pyenv-init - bash
+  PYENV_INSTALL_PATH="${exec_root}" run pyenv-init - bash
```

Comment:
Looks like we rather need to remove `exec_root` altogether in favor of `PYENV_INSTALL_PATH` (same elsewhere)